js/ui/date_box/ui.date_box.base.js

```diff
@@ -147,6 +147,13 @@ const DateBox = DropDownEditor.inherit({
              */
             disabledDates: null,
 
+            /**
+             * @name dxDateBoxOptions.navigateToSameDate
+             * @type boolean
+             * @default true
+             */
+            navigateToSameDate: true,
```

Comment:
Should we add this option to the DateBox? We can change this option with [calendarOptions](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#calendarOptions)

Reply:
> Should we add this option to the DateBox? We can change this option with [calendarOptions](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#calendarOptions)

This option is useful the most when using 'disabledDates' option together. 'disabledDates' option can be used only in dateBox configuration. So I made by analogy. Check it: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#calendarOptions

Reply:
@AlexJameson  What do you think?
This option specific for the `calendar` pickerType only. I'll prefer to use `calendarOptions` instead. Perhaps we should consider deprecating `disabledDates` (as it was done with [maxZoomLevel](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#maxZoomLevel))

Reply:
@DokaRus 
I think that moving this option to `calendarOption` is a good idea if you can use it **only** with the calendar picker type.

Reply:
The option is moved to calendarOptions